src/modules/mount/mount.conf

```diff
@@ -108,15 +108,15 @@ btrfsSwapSubvol: /@swap
 #
 # mountOptions:
 #    - filesystem: default
-#      options: [ defaults, noatime ]
+#      options: [ defaults ]
 #
 mountOptions:
     - filesystem: default
-      options: [ defaults, noatime ]
+      options: [ defaults ]
     - filesystem: efi
       options: [ defaults, umask=0077 ]
     - filesystem: btrfs
-      options: [ defaults, noatime, compress=lzo ]
+      options: [ defaults, compress=zstd:1 ]
     - filesystem: btrfs_swap
       options: [ defaults, noatime ]
```

Comment:
I'm pretty sure `noatime` is meaningless for swap, as it's not actually a filesystem

Reply:
btrfs_swap isn't a swap partition.  It is the subvolume that holds the btrfs swapfile.